README.md

```diff
@@ -1069,9 +1069,15 @@ yet but might provide some value.
   end
   ```
 
-### Style Guides
+### Alternative Style Guides
 
-Check [Awesome Elixir][Style Guides] for a list of alternative style guides.
+* [Aleksei Magusev's Elixir Style Guide]
+  — An opinionated Elixir style guide. Developed by some of the Elixir's core
+  members and used in the Elixir project itself.
```

Comment:
I think we can say "the library closest to the Elixir project itself" or something like this.